src/strings/csv/es.csv

```diff
@@ -386,6 +386,9 @@ GRAMINOID,Gramínea
 GRAMS,Gramos
 GREATER_THAN_FIFTY_PERCENT,> 50%
 GROWTH_FORM,Forma de crecimiento
+HIGHEST,Más alta
+HIGHEST_AND_LOWEST_MORTALITY_RATE_CARD_TITLE,¿Qué zonas tiene la tasa de mortalidad más alta y más baja?
```

Comment:
@constanzauanini Alex updated this translation.. looks good now I think, yes?

Reply:
No, it should be "¿Qué zonas tienen ... ", with N beacuse it's plural

Reply:
Ah ok, I think we updated the wrong string. Can you work with @alexTerraformation on fixing the right one and reverting this?